### ISSUE TITLE
Cluster apply fails if not preceded with an ssh to bastion/cluster instance

### ISSUE THREAD

**Issue by charlieegan3** [jetstack/tarmak#174]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
I was creating a pipeline to tarmak cluster apply. I found that I needed to tarmak cluster ssh bastion before as otherwise the ssh failed when connecting to the bastion during the apply.

**What you expected to happen**:
I'd expect to be able to apply without sshing first.

**How to reproduce it (as minimally and precisely as possible)**:
In a tarmak-config repo, from a clean checkout / working tree. Run a cluster apply against the non-hub cluster.

**Comment by simonswine** [comment]
This is the bug I was talking to you about @JoshVanL 

I would say steps to reproduce:

- Given you have a multi cluster environment dev with cluster green
- You apply dev-hub
- You clean out everything in your tarmak config (esp. dev-hub, dev-green folders)
- You try to apply dev-green, then it fails generate ssh-config dev-hub (it fails for tarmak provider inside terraform run) 

Error should be something like that:

```
time="2018-04-18T10:47:57Z" level=debug msg="* tarmak_vault_cluster.vault: call to Tarmak.VaultClusterStatus failed: failed to initialise vault cluster: time out verifying that vault cluster is initialiased and unsealed: %!s(<nil>)" app=tarmak module=terraform std=err
```

/assign @JoshVanL

**Comment by simonswine** [comment]
Fixed through #247
